examples/with-gsap/components/Content.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useRef } from 'react'
+import { gsap } from 'gsap'
+
+const Content = () => {
+  let line1 = useRef(null)
+  useEffect(() => {
+    gsap.from([line1], 0.6, {
+      delay: 0.9,
+      ease: 'power3.out',
+      y: 24,
+      stagger: {
+        amount: 0.15,
+      },
+    })
+  }, [line1])
+
+  return (
+    <p ref={(el) => (line1 = el)} className="line">
+      A Simple example using{' '}
+      <a
+        href="https://greensock.com/gsap/"
+        style={{ fontWeight: 'bold', textDecoration: 'none' }}
+      >
+        GSAP
+      </a>{' '}
+      &{' '}
+      <a
+        href="https://www.npmjs.com/package/react-transition-group"
+        style={{ fontWeight: 'bold', textDecoration: 'none' }}
+      >
+        react-transition-group
+      </a>
+    </p>
+  )
+}
```

Comment:
The `useRef` doesn't seem to be used correctly across all these examples:

1. The returned reference can be passed in directly, it should not be reassigned as that defeats the purpose
2. The current reference should be accessed as `line1.current` once point no 1 above is addressed